ADRs/ADR_TEMPLATE.md

```diff
@@ -1,72 +1,107 @@
-# [short title of solved problem and solution]
+# [short title of solved problem and solution] <!-- omit in toc -->
 
-* Status: [proposed | rejected | accepted | deprecated | … | superseded by [ADR-0005](0005-example.md)] <!-- optional -->
-* Deciders: [list everyone involved in the decision] <!-- optional -->
-* Date: [YYYY-MM-DD when the decision was last updated] <!-- optional -->
+- Status: [proposed | in review | rejected | accepted | deprecated | … | superseded by [ADR-0005](0005-example.md)]
```

Comment:
Please review #36 when you get a chance, I've updated the statuses there as well as included the status flowchart in the README.

TLDR; these are the statuses I proposed:
```
draft | in review | changes needed | withdrawn | accepted | superseded
```